modules/local/whatshap.nf

```diff
@@ -0,0 +1,19 @@
+process WHATSHAP {
+
+    input:
+    tuple val(meta), path(reads), path(read_index)
+    tuple val(meta3), path(reference)
+    tuple val(meta4), path(reference_index)
+    tuple val(meta5), path(vcf)
```

Comment:
I get that this is based on the nf-core way of doing things, but these `meta`s could probably be left out.